.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Apply Ops Dashboard CI
+
+on: [push, pull_request] 
+
+jobs:
+  linting:
+    name: Linting
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: 'Checkout Code'
+
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: .ruby-version
+
+    - name: Install dependencies
+      run: bundle install
+    - name: Run Rubocop
+      run: bundle exec rubocop
+  
+  rspec:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: 'Checkout Code'
+    
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: .ruby-version
+    
+    - name: Install dependencies
+      run: bundle install
+    
+    - name: Run Tests
+      env:
+        AZURE_ACCESS_TOKEN: ${{ secrets.AZURE_ACCESS_TOKEN }}
+        AZURE_USERNAME: ${{ secrets.AZURE_USERNAME }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We shouldn't have to add secrets because we're not making real requests. To avoid having to do this we can add a `.env.test` file.

Reply:
Fixed in https://github.com/DFE-Digital/apply-ops-dashboard/pull/23https://github.com/DFE-Digital/apply-ops-dashboard/pull/23